library/ns1_zone.py

```diff
@@ -135,28 +168,26 @@
 
 
 try:
-    from ns1 import NS1, Config
     from ns1.rest.errors import ResourceException
 except ImportError:
     # This is handled in NS1 module_utils
     pass
```

Comment:
just wanna make sure this except is still needed - similar but different enough from the one above that i just wanna make sure it's still "load-bearing"

Reply:
Yup, still needed.  The reason this `except` is here is because we want to throw a graceful failure via Ansible when the user is missing a required lib and not just throw a python error.

Since this code will be required in every module, we've made this DRY by also catching the [ImportError](https://github.com/ns1/ns1-ansible-modules/blob/396abc160f4473572c9996d2f182ff84e9e711b0/module_utils/ns1.py#L25) in our module_util and handling the failure there.

This is a pattern I stole from Ansible core modules, but I'm down to improve this if you have any idea?

Reply:
nope, just wanted to make sure it wasn't dangling and didn't spare the cycles to find out for myself :)